### DISCUSSION TITLE
On this day feature

### DISCUSSION THREAD

**Discussion by zigatilli** [deckerst/aves#55] [Ideas]
On this day feature (memories), like google photos or snapchat.

**Comment by deckerst** [comment]
Interesting idea. I get the concept, but I'm not a user of this feature myself, so I'm not familiar with the way they implement it.

I took a quick look at Google Photos:
- they surface pictures for a week, not a day
- pictures are presented in a kind of slideshow
- the feature is combined with other automatic creations (like collages)
- customization prevents surfacing specific people, dates, etc.

And I *guess* that they also:
- avoid visually close duplicates
- avoid non-memorable categories of photos (like pictures of receipts and other documents)
- prefer pictures with recurring people, like family and friends?

So, to do this right in Aves, I'd need to tackle first: ML features for picture categorization, face detection, duplicate detection, and then some kind of slideshow or diary-style presentation. In other words, it will take a while.

@zigatilli One thing that would help is for you (and other users of this feature) to let me know what you like about that feature. Is it showing you pictures of events and parties? Family and friends? Trips out of your usual area?
